static/js/message_viewport.js

```diff
@@ -30,8 +30,8 @@ function make_dimen_wrapper(dimen_name, dimen_func) {
     };
 }
 
-export const height = make_dimen_wrapper("height", $($message_pane).height);
-export const width = make_dimen_wrapper("width", $($message_pane).width);
+export const height = make_dimen_wrapper("height", $.fn.height);
+export const width = make_dimen_wrapper("width", $.fn.width);
```

Comment:
(`$message_pane` is always `undefined` at this point since we haven’t yet called `initialize()`, and has been since 2014: 198648447ff93afe517fe8abbcb255ba59c6bc9e. The only reason this does anything is that `make_dimen_wrapper` uses `.call` and `.apply` to replace the `this` binding for these `height` and `width` methods. So we might as well stop pretending that they’re bound until then.)